packages/gatsby-theme-styleguide/src/gatsby-components/SiteMapData/tools.js

```diff
@@ -0,0 +1,69 @@
+/*
```

Comment:
For this function, generally speaking I would say that it seems to be good, but your choices of word to relate to things make it hard to understand. 

Either use comments, or preferably try to choose variable, arguments and function names that make your code self documenting, you don't want the next developper to pull hairs of his head trying to understand what you did try to achieve.

Do not hesitate to have longer names if they convey meaning.